src/Paramore.Brighter/CodeAnalysis/MemberNotNullWhenAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿#if NETSTANDARD
+namespace System.Diagnostics.CodeAnalysis;
+
+[AttributeUsage(AttributeTargets.Method | AttributeTargets.Property, Inherited = false, AllowMultiple = true)]
+public sealed class MemberNotNullWhenAttribute : Attribute
```

Comment:
Out of interest, we have an open conversation on using polyfills vs dropping netstandard20

Reply:
I tend to prefer drop support for `netstandard20`, but depending on how many downloads we have today on .NET standard maybe we still have people using Brighter with .NET Framework 4

Maybe on V11 we can remove the support for .NET Standard completed, and put a warning on V10 saying that is the last version of supporting .NET Standard/Framework

Reply:
Yeah, you may be right that it would be 0-warning if we dropped it with V10 and we should declare that will happen with V11.